### DISCUSSION TITLE
Consider dropping the overpaid tag when fees are low

### DISCUSSION THREAD

**Discussion by knorrium** [mempool/mempool#816] [Ideas]
The overpaid tag should probably be dropped when the fees are below a certain threshold. There's not a lot of room for lowering the fee and the experience ends up being a bit confusing.

![04998A36-82A9-4017-9B0C-31AA77D4E5BF](https://user-images.githubusercontent.com/100320/126890455-088961ac-08a6-405a-9160-7a81d46af4c3.jpg)

**Comment by softsimon** [comment]
Well, you could probably have paid 1 sat/vB to get into that block, and you overpaid double .. so it's correct. But I see your point .. not much difference from 1 to 5 sats .. fromm 100 to 200 is much bigger. So hiding the tag for anything under 5 or 10 sats is an idea.

**Comment by knorrium** [comment]
hiding when it's under 10 seems reasonable

**Comment by softsimon** [comment]
> hiding when it's under 10 seems reasonable

Note also that as the bitcoin dollar price goes up, the amount of sats per byte people are paying will go down. 10 sat/vB today is much more expensive than just 4 years ago. (If I am thinking of this correctly)
